irohad/validation/stateful_validator_common.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_STATEFUL_VALIDATOR_COMMON_HPP
+#define IROHA_STATEFUL_VALIDATOR_COMMON_HPP
+
+#include "common/types.hpp"
+
+namespace shared_model {
+  namespace interface {
+    class Proposal;
+  }
+}
+
+namespace iroha {
+  namespace validation {
+
+    /// Name of the failed command
+    using CommandName = std::string;
+
+    /// Error, with which the command failed
+    using CommandError = std::string;
+
+    /// Failed command's name and error
+    using CommandNameAndError = std::pair<CommandName, CommandError>;
```

Comment:
Why isn't it a result?

Reply:
Why should it be a result? It can't be success or fail, it's just a structure for name and error itself